lib/iceserversource/nts.js

```diff
@@ -112,7 +116,13 @@ class NTSIceServerSource extends EventEmitter {
         get() {
           return !!this._currentPoll;
         }
-      }
+      },
+      status: {
```

Comment:
@markandrus Can we use a getter method here?

Reply:
Yes, good idea 👍